src/options-pages/fuzzer.cpp

```diff
@@ -43,13 +44,13 @@ Fuzzer::Fuzzer(Settings::FuzzerPtr settings)
     setCategory(Constants::SETTINGS_CATEGORY);
 }
 
-bool Fuzzer::matches(const QString &searchKeyWord) const
+bool Fuzzer::matches(const QRegularExpression &regexp) const
 {
     const QStringList keywords{"asn1", "asn.1", "acn", "test", "fuzzer", "asn1scc"};
     for (const auto &keyword : keywords)
-        if (keyword.contains(searchKeyWord, Qt::CaseInsensitive))
+        if (keyword.contains(regexp.pattern(), Qt::CaseInsensitive))
```

Comment:
well... it should be probably the other way around - regexp should be checked for matching with any of the keywords

Reply:
done